pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -140,6 +143,18 @@ func New(chain Ledger, cfg config.NeoFSBlockFetcher, logger *zap.Logger, putBloc
 	if err != nil {
 		return nil, err
 	}
+
+	getCfg := chain.GetConfig()
+	m := smartcontract.GetDefaultHonestNodeCount(int(getCfg.ValidatorsCount))
+	baseHeaderSize := block.New(chain.GetConfig().StateRootInHeader).GetExpectedHeaderSize() - 2
+	vs, _ := keys.NewPublicKeysFromStrings(getCfg.StandbyCommittee)
+	verification, _ := smartcontract.CreateDefaultMultiSigRedeemScript(vs[:getCfg.GetNumOfCNs(0)])
+	defaultWitness := transaction.Witness{
+		InvocationScript:   make([]byte, 66*m),
+		VerificationScript: verification,
+	}
+	w := gio.NewBufBinWriter()
+	defaultWitness.EncodeBinary(w.BinWriter)
```

Comment:
You don't need to do it manually. Fill dummy witness and use `GetExpectedHeaderSize`.